lib/licenses.nix

```diff
@@ -152,6 +152,11 @@ lib.mapAttrs mkLicense (
       fullName = "Artistic License 2.0";
     };
 
+    asl = {
+      fullName = "Academic Software License";
+      url = "https://github.com/ACEsuit/ACE.jl/blob/main/ASL.md";
+    };
+
```

Comment:
Given we already have `asl20`, I'm not sure calling a completely different license `asl` is a good idea…

Reply:
Also the licence is not FOSS and must be marked `free = false;` (it is not even redistributable AFAICT, since all the grants are conditional on non‐commercial use).

Reply:
Is it likely to be used for more than just this one package?